### PR TITLE
Add prettier and upgrade eslint

### PR DESCRIPTION
**Pull request by sthomas1618**

### CODE REVIEW

File: src/select/SimpleSelect.js [hunk 1]
```diff
@@ -578,68 +564,62 @@ class SimpleSelect extends Component {
...
-    const ClearComponent = this.props.clearComponent
-    const {
-      clearable,
-      clearRenderer,
-      disabled,
-      value
-    } = this.props
+    const { clearComponent } = this.props
+    const ClearComponent = clearComponent
+    const { clearable, clearRenderer, disabled, value } = this.props
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:584]: super nit: could combine lines 582 and 584

File: src/select/SimpleSelect.js [hunk 2]
```diff
@@ -578,68 +564,62 @@ class SimpleSelect extends Component {
...
+        disabled={disabled}
+        onClearClick={this.onClearClick}
+        onClearTouchEnd={this.onClearTouchEnd}
+      />
+    )
   }
 
   renderLoading() {
-    const LoadingComponent = this.props.loadingComponent
+    const { loadingComponent } = this.props
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:602]: same nit: I would combine the `this.props` lines

File: src/select/SimpleSelect.js [hunk 3]
```diff
@@ -668,65 +648,73 @@ class SimpleSelect extends Component {
...
+    const { ariaLiveSelection, isFocused } = this.state
+
+    if (!isFocused) {
+      return null
+    }
+
     return (
       <div className="sr-only" aria-live="polite">
-        <p id="aria-selection-event">&nbsp;{this.state.ariaLiveSelection}</p>
-        <p id="aria-context">&nbsp;{this.constructAriaLiveMessage()}</p>
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:675]: I wonder why those spaces were there...that seems oddly specific 🤔

File: src/select/ValueGroup.js [hunk 4]
```diff
@@ -4,12 +4,6 @@ import PropTypes from 'prop-types'
 import { getSelectValue } from './utils'
 
 class ValueGroup extends Component {
-  onMouseDown = (event) => {
-    if (this.props.onMouseDown) {
```
Comment (vicky-holcomb) [inline src/select/ValueGroup.js:8]: Why remove this, was it not used?

**Review by vicky-holcomb — APPROVED** [review]

File: src/select/SimpleSelect.js [hunk 5]
```diff
@@ -668,65 +648,73 @@ class SimpleSelect extends Component {
...
-      open: isOpen, focus: isFocused, 'outer-focus': isOuterFocused, disabled
+    const selectClassName = classNames('crane-select', className, {
+      open: isOpen,
+      focus: isFocused,
+      'outer-focus': isOuterFocused,
+      disabled
     })
 
     return (
-      <div aria-expanded={isOpen} className={selectClassName} ref={(container) => { this.container = container }}>
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:729]: Hmm why remove this ref?

File: src/select/SimpleSelect.js [hunk 6]
```diff
@@ -649,7 +629,7 @@ class SimpleSelect extends Component {
     const hasStaticOptions =
       (Array.isArray(staticOption) && !!staticOption.length) || !!staticOption
     if (opts.length || hasStaticOptions) {
-      const MenuComponent = this.props.menuComponent
+      const MenuComponent = menuComponent
       menu = (
         <div className="crane-select-menu-container" role="listbox">
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:634]: I think to fix the a11y issue we want to remove this role and add `id='crane-select-menu-container`

Comment (vicky-holcomb) [reply src/select/SimpleSelect.js:634]: Also to test I think you'll have to use a select component that allows filtering and the menu has to be open when you run the a11y tests.

File: src/select/ValueGroup.js (hunk 4 above)
Comment (sthomas1618) [reply src/select/ValueGroup.js:8]: Nope, after upgrading eslint, it flagged it as not used. I couldn't find any usages of it 🤷🏽

File: src/select/SimpleSelect.js (hunk 5 above)
Comment (sthomas1618) [reply src/select/SimpleSelect.js:729]: Eslint flagged it and I looked and could not find any usages.

File: src/select/SimpleSelect.js (hunk 1 above)
Comment (sthomas1618) [reply src/select/SimpleSelect.js]: Fixed!

File: src/select/SimpleSelect.js (hunk 2 above)
Comment (sthomas1618) [reply src/select/SimpleSelect.js]: Fixed!

File: src/select/SimpleSelect.js (hunk 6 above)
Comment (vicky-holcomb) [reply src/select/SimpleSelect.js:641]: @sthomas1618 did this one get fixed?

**Pull request merged**